Casamari/MNC010.xml

```diff
@@ -56,7 +56,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                   </msItem>
                   <msItem xml:id="ms_i3">
                      <locus from="34r" to="35v"/>
-                     <title type="complete" ref="LIT4962Basema">‘In the name of God, the First One without yesterday’</title>
+                     <title type="complete" ref="LIT2993RepCh274">‘In the name of God, the First One without yesterday’</title>
```

Comment:
I am currently working on this Ms, and changed the whole ms content, 
Is it needed to change it here? 
I am afraid of competing changes.

Reply:
There might be a conflict, which will be very easy to solve, because this change is minimal

Reply:
So, why not leaving out this manuscript?

Reply:
If your manuscript will be ready very soon, I can revert the change, but I am rather reluctant out of principle to knowingly use deleted IDs